src/app/shell/Header.tsx

```diff
@@ -306,8 +328,38 @@ class Header extends React.PureComponent<Props, State> {
     if (!prevState.showSearch && this.state.showSearch && this.searchFilter.current) {
       this.searchFilter.current.focusFilterInput();
     }
+
+    this.updateVendorEngrams(this.props.account || undefined);
```

Comment:
Check to see if the destiny version of the account has changed, and only reset if that's the case.

Reply:
Unless accounts are changed while we're waiting for VE.xyz to resolve a fetch, updating will just look at cached data.